### DISCUSSION TITLE
removable or modular features

### DISCUSSION THREAD

**Discussion by sativarg-tuta** [ByteWelder/Decktility#4] [Ideas]
Hello Ken  and or community,
I have seen some modular PC ideas that could help blind users. Some people could use a PC like yours without a screen... some have learned to use just six keys to type. I could see a PC like yours without a visual element lasting a long time on a very compact power source that could be very nice for blind and sighted people as well. Special gloves could allow for hands free typing and blue tooth devices could do some interesting things too?
People skilled with screen reader tech can do wonders without power hungry displays.
Ever hear of Accessible-Coconut  Linux?
• Six key inputs: Regular keyboards are often hard to use for visually challenged people. This feature helps users type any form of text using six keys of the keyboard with Braille knowledge in Perkin's Brailler mode. 
Could less be much much more?
Units like this could find uses in special situations like in Earth orbit, on the battle field or on patrol?
thanks for reading and considering...
blessings
chuck 🔥💖🔥

**Comment by KenVanHoeylandt** [comment]
Hi Chuck! I didn't consider accessibility up to this point, but the good news is that the keyboard is already built in a modular way. I'm working on a better (regular) keyboard currently. An A11Y-focused one is something I'll consider in the future. Do you have any A11Y keyboard examples or layouts that I should consider in the future?

The screen also has hardware buttons on the back, allowing you to turn off the backlight entirely. This should conserve at least _some_ battery power.

For usage outdoors, one would probably want a much more rigid device with at least some level of water-proofing. This is outside of my current skill level and means.

I didn't know of Accessible Coconut yet. It looks interesting.

**Comment by sativarg-tuta** [comment]
Hi Ken,
Thank you for your kind reply and interest.
I am ashamed to say that I do not have the brail typing skill. In fact I just misspelled brail?
Ever since I went blind I have been watching for a laptop with removable or modular display. For me, it is a waist of power to even have it onboard. I think every modern PC, laptop and phone should be capable of being worked on without display. The bios should talk so that anyone can set it up if the display goes bad?
Simple speech menus do not take up that much memory considering how small all things are now?
I am big on ideas and talk and not so great on skill... sorry.
I never learned brail. or is it brale? No? spelling sucks, or is it me? LOL
Thanks for hearing me
blessings
chuck 🔥💖🔥
LOL OK Braille? ? ?

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



May 29, 2023, 10:20 AM by ***@***.***:

**Reply by KenVanHoeylandt** [reply]
The problem with removing the screen is that the build is using a Pi Compute Module 4: The unit with the screen also holds the Pi and all the peripheral connectors. There isn't much to do in that regard unless one would make an entirely different device.

Meanwhile, I've been thinking: it might be a good option to have some kind of keyboard cover that only houses a USB connector and some magnetic contacts. If there exist small braille devices out there, they could then be snapped on magnetically. I'll research some braille readers later.

**Comment by sativarg-tuta** [comment]
Thank you Ken,
I should have seen that removing the screen was impractical.
 Braille interface is interesting as a stealth or silent asset?
Some situations could benefit like a space walk or covert mission?
Is that too weird?
thanks again...

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



May 29, 2023, 10:49 AM by ***@***.***:
